src/plugins/discover/public/application/main/components/sidebar/discover_field_search.tsx

```diff
@@ -272,30 +344,51 @@ export function DiscoverFieldSearch({ onChange, value, types }: Props) {
         </EuiFlexItem>
       </EuiFlexGroup>
       <EuiSpacer size="xs" />
-      <EuiOutsideClickDetector onOutsideClick={() => {}} isDisabled={!isPopoverOpen}>
-        <EuiFilterGroup className="dscFieldSearch__filterWrapper">
+      <EuiFlexItem>
+        <EuiFilterGroup fullWidth>
+          <EuiOutsideClickDetector onOutsideClick={() => {}} isDisabled={!isPopoverOpen}>
+            <EuiPopover
```

Comment:
I think there's some extraneous, very old leftover wrappers here that aren't necessary anymore
```suggestion
        <EuiFilterGroup fullWidth>
          <EuiPopover
```